invisible_cities/core/exceptions.py

```diff
@@ -0,0 +1,7 @@
+"""
+Define IC-specific exceptions
+"""
+
+class NoInputFiles(Exception):
+    """ Input files list is not defined """
+    pass
```

Comment:
Nice.

Could we please make this a subclass of `InvisibleCitiesException` or `ICException`.

Our exception hierarchy will require a but more thought, but this is fine for now.